docs/setup.rst

```diff
@@ -0,0 +1,95 @@
+Setting Up Corobo
+=================
+
+The following documentation illustrates the process to
+install corobo and interface it with the Gitter 
+(or any other backend) API.
+
+Installation
+------------
+- To install corobo, first you need to install 
+  python (version 3.6 or more) and set up a virtual environment. 
+  This can be done as follows:
+  ::
```

Comment:
This is incorrect, you should keep this at the end of the paragraph above. Github somehow renders this but I wasn't able to get this rendered properly locally.